<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEleMainWayDialog</class>
 <widget class="QDialog" name="CEleMainWayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>6</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u_a">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>6</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A相电压</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>34</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u_b">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>34</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B相电压</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>60</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C相电压</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u_c">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>6</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kW</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_active_a">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>6</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A相有功功率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>34</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kW</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_active_b">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>34</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B相有功功率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>60</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kW</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C相有功功率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_active_c">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>90</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kW</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>总有功功率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_active_all">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>10</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_frequence">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_56">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>频率</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_power_factor_a">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>6</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A相功率因数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_factor_b">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>34</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B相功率因数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C相功率因数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_factor_c">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>总功率因数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_factor_all">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>6</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kvar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_reactive_a">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>6</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A相无功功率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>34</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kvar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_reactive_b">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>34</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B相无功功率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>60</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kvar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_46">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C相无功功率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_reactive_c">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>86</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kvar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_48">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>86</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>总无功功率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_reactive_all">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>86</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_49">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>6</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kWh</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_energy_active">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>6</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>总有功电能</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_53">
    <property name="geometry">
     <rect>
      <x>212</x>
      <y>34</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kvarh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_54">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>总无功电能</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_energy_reactive">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>34</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_59">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>6</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kVA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_apparent_a">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>6</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_60">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A相视在功率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_61">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>34</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kVA</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_apparent_b">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>34</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_62">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B相视在功率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_63">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kVA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_64">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C相视在功率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_apparent_c">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_65">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>86</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>kVA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_66">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>86</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>总视在功率</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_power_apparent_all">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>86</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_8" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_67">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>6</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u_ab">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>6</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_68">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AB线电压</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_69">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>34</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u_bc">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>34</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_70">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>BC线电压</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_71">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>60</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_72">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CA线电压</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_u_ca">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_9" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_73">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>6</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i_a">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>6</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_74">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>A相电流</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_75">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>34</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i_b">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>34</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_76">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>B相电流</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_77">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>60</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_78">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>C相电流</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i_c">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_79">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>86</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_80">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>86</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>零序电流</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_i_zero">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>86</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_next_page">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return_back">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_pre_page">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
